sos/plugins/openvswitch.py

```diff
@@ -43,6 +43,16 @@ def setup(self):
             "/var/run/openvswitch/ovs-vswitchd.pid"
         ])
 
+        # Gather fdb statistical output for each OVS bridge on the host.
+        br_list_result = self.call_ext_prog("ovs-vsctl list-br")
+        if br_list_result['status'] == 0:
+            for br in br_list_result['output'].splitlines():
+                # FDB statistics command, first iteration do this early on
+                # so we get some natural delay between the two iterations.
+                self.add_cmd_output(
+                    ["sh -c \"ovs-appctl fdb/stats-show %s && date +%%T.%%3N\"" % br],
+                    suggest_filename="ovs-appctl fdb/stats-show %s  seq1" %br)
```

Comment:
Nack. We don't use the shell in `sos` unless it is completely unavoidable. There is no need to datestamp any of the data collected by sos, since the inode timestamps and logs already provide this information in multiple forms.

This also means you can get rid of `suggest_filename` - this is really intended to be used only when the automatically generated name is unwieldy, or exceeds file system name length limits.

Reply:
There's no need to wrap a single command in a list to pass to `add_cmd_output()` - if you look at the docs, or the source, you'll see that it accepts either a single string, or a list of strings.

Reply:
I'm using the suggest_filename here because the same command is executed twice. I will change it and only use it the second run.